docs/baking.md

```diff
@@ -0,0 +1,185 @@
+download the large `sprout-groth16.params` which is unused at runtime.
+So, it's better to download them to the `/usr/local/share/zcash-params`, the easiest way to do that
+is to run [`fetch-sapling-params.sh`](../scripts/fetch-sapling-params.sh) script:
+```
+sudo ./fetch-sapling-params.sh
+```
+
+Sudo is required because the files are downloaded to the `/usr/local/share/zcash-params`.
+This script also checks that hashsums of the downloaded files match the expected values.
+
+## Setting up the tezos-node
+
+The easiest way to set up a node running on the `mainnet` or on one of the
+testnets (e.g. `delphinet` or `edonet`) is to use one of the predefined
+`tezos-node-<network>.services` systemd services provided in the `tezos-node`
+package. 
+
+However, by default, these services will start to bootstrap the node from scratch,
+which will take a significant amount of time.
+In order to avoid this, here we suggest bootstrapping from a snapshot instead.
+
+### Setting up node data directory
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as its RPC address by default.
+These parameters can be changed in the service configuration, which is located in
+`/lib/systemd/system/tezos-node-<network>.service`. Note, that if you've updated the `.service`
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
+
+### Bootstrapping the node
+
+In order to run a baker locally, you'll need a fully-synced local `tezos-node`.
+
+The fastest way to bootstrap the node is to import a snapshot.
+Snapshots can be downloaded from https://snapshots.tulip.tools/#/ .
+
+All commands within the service are run under the `tezos` user.
+
+Download the snapshost for desired network and with the desired history mode
+(you can read more about `tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+Snapshot import requires that the `tezos-node` data directory doesn't contain the `context` and `store` folders.
+Remove them before the snapshot import.
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+In case you're getting an error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do that run one of the following commands
+depending on whether the config in the given data directory was initialized previously:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
```

Comment:
is it possible to avoid having to specify `--data-dir` in all the commands? isn't it coming from tezos node config file? isn't there a default?

Reply:
AFAIU, in case when  `--data-dir` isn't presented it'll use `$HOME/.tezos-node` by default. It's possible to specify `--config-file` instead of `--data-dir`, but you'll still have to provide at least one of these options

Reply:
these commands are to be executed as  `tezos` user, who's $HOME directory should be set to `/var/lib/tezos` and then defaults work just fine; also, a new user on a happy path/fast track to running node and baking on `mainnet` - which I think is the primary use case - shouldn't need to do any of this at all because `mainnet` is the default. To simplify things further (for the user, not necessarily for the packager 😈 ) `config init` should probably run at package installation time, so that user only ever needs to run `config update`, and that also becomes a note - explanation of how to set up for a testnet instead of mainnet - rather than the main story line.

Reply:
`config init` call is now a part of `tezos-node` package postinsallation

Reply:
There is now a problem with this, which is that the `--network` option does not work for `edo2net`, which I think will be somewhat common considering it's the only other network presented aside from `mainnet`